pkg/gatewayserver/io/ws/lbslns/upstream_test.go

```diff
@@ -214,7 +214,7 @@ func TestJoinRequest(t *testing.T) {
 				RxMetadata: []*ttnpb.RxMetadata{
 					{
 						GatewayIds:  &gtwID,
-						Time:        &[]time.Time{time.Unix(1548059982, 0)}[0],
+						Time:        ttnpb.ProtoTime(&[]time.Time{time.Unix(1548059982, 0)}[0]),
```

Comment:
These were some old hacks that we had in place since you cannot take pointer of a value returned by a function.

Can be significantly simplified to `ttnpb.ProtoTimePtr(time.Unix(1548059982, 0))`. Same bellow.